src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -340,6 +337,9 @@
   <data name="CmdProfileArgDesc" xml:space="preserve">
     <value>Open with the given profile. Accepts either the name or GUID of a profile</value>
   </data>
+  <data name="CmdSessionIdArgDesc" xml:space="preserve">
+    <value>Sets the WT_SESSION variable</value>

```

Comment:
```suggestion
    <value>Sets the WT_SESSION variable; must be a GUID</value>
```


maybe? or, is there an opportunity for like, yeeting an environment variable injection into this arg?